aws/resource_aws_db_cluster_snapshot.go

```diff
@@ -167,6 +170,24 @@ func resourceAwsDbClusterSnapshotRead(d *schema.ResourceData, meta interface{})
 	d.Set("storage_encrypted", snapshot.StorageEncrypted)
 	d.Set("vpc_id", snapshot.VpcId)
 
+	if err := saveTagsRDS(conn, d, aws.StringValue(snapshot.DBClusterSnapshotArn)); err != nil {
+		log.Printf("[WARN] Failed to save tags for RDS DB Cluster Snapshot (%s): %s", d.Id(), err)
+	}
+
+	return nil
+}
+
+func resourceAwsdbClusterSnapshotUpdate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).rdsconn
+
+	if d.HasChange("tags") {
+		if err := setTagsRDS(conn, d, d.Get("db_cluster_snapshot_arn").(string)); err != nil {
+			return err
+		} else {
```

Comment:
Nit: Since the `if` condition calls `return`, the `else` and nesting are extraneous 👍